content/post/2020-10-14-dv-signal.Rmd

```diff
@@ -0,0 +1,259 @@
+
+# Set a bunch of fields so that the data frames know how to plot themselves
+df_dv_avg$time_value = df_in_avg$time_value = start_day
+df_dv_avg$issue = df_in_avg$issue = start_day
+attributes(df_dv_avg)$geo_type = attributes(df_in_avg)$geo_type = "state"
+class(df_dv_avg) = c("covidcast_signal", class(df_dv_avg))
+class(df_in_avg) = c("covidcast_signal", class(df_in_avg))
+
+# Plot choropleth maps, using the covidcast plotting functionality
+subtitle = paste("Averaged over", start_day, "to", end_day)
+p1 = plot(df_dv_avg,
+          title = "% COVID-like illness, DV signal",
+          range=c(0, 20), choro_params = list(subtitle = subtitle))
+p2 = plot(df_in_avg,
+          title = "Daily new confirmed COVID-19 cases per 100k pop.",
+          range=c(0, 30), choro_params = list(subtitle = subtitle))
+grid.arrange(p1, p2, nrow = 1)
+```
+
+### Backfill
+Electronic medical records have a few interesting challenges not present in the survey data. First, is that the data is available at a significant and variable latency (we call this "backfill"). For claims related to a given day, we will receive only about half of them within the first seven days after that date. Due to backfill, we cannot make estimates of our indicator for a given day until four days have passed, and we will revise the value of that day's indicator every day.
+Interestingly, certain types of claims have a longer latency than others. We found that for a given date, as more claims are available, %CLI usually increases. Covid-like claims tend to have a longer latency and thus our early estimates of the indicator usually underestimate the final value.
+
+### Weekday Effects
+Another challenge is the influence of the day of the week on %CLI. On weekends, both total counts and COVID-like counts decrease, but proportionally, total counts decrease more. This is because people are more likely to delay seeking medical care for non-acute issues than acute issues. The total counts include many visits related to non-acute issues, but almost all COVID-like counts are due to acute issues. Without correcting for this weekday effect, the %CLI curve has a "sawtooth" pattern, spiking on weekends. We derived a method to create a corrected signal that accounts for this weekday effect (for a formal mathematical description of the method, see our [API documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/doctor-visits.html#day-of-week-adjustment)). Below, we see the effect of making these corrections below. When we do not correct for the weekday effect, we see a sawtooth pattern that clearly does not represent true changes in COVID-like illness within a location. However, when we make the weekday correction, we get a smooth curve that looks reasonable. It is important to note that this correction is not time-wise smoothing! Rather, we are making a point-wise correction based on the fitted $\alpha$ parameters.
```

Comment:
```suggestion
Another challenge is the influence of the day of the week on %CLI. On weekends, both total counts and COVID-like counts decrease, but proportionally, total counts decrease more. This is because doctor visits during the weekend tend to focus on acute care.  The total counts include many visits related to non-acute issues, but almost all COVID-like counts are due to acute issues. Without correcting for this weekday effect, the %CLI curve has a "sawtooth" pattern, spiking on weekends. We derived a method to create a corrected signal that accounts for this weekday effect (for a formal mathematical description of the method, see our [API documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/doctor-visits.html#day-of-week-adjustment)). Below, we see the effect of making these corrections. When we do not correct for the weekday effect, we see a sawtooth pattern that clearly does not represent true changes in COVID-like illness within a location. However, after making the weekday correction, we get a smooth curve that looks reasonable. It is important to note that this correction is not time-wise smoothing! Rather, we are making a point-wise correction based on seven fitted parameters, one for each day of the week.
```